src/enable_variable_ops/ngraph_tracked_variable.cc

```diff
@@ -283,19 +283,6 @@ void NGraphVariableOp::Compute(OpKernelContext* ctx) {
   ngraph::Event::write_trace(event_compute);
 }
 
-REGISTER_OP("NGraphVariable")
-    .Output("ref: Ref(dtype)")
-    .Attr("shape: shape")
-    .Attr("dtype: type")
-    .Attr("just_looking: bool = false")
-    .Attr("is_tf_just_looking: bool = false")
-    .Attr("copy_to_tf: bool = false")
-    .Attr("container: string = ''")
-    .Attr("shared_name: string = ''")
-    .Attr("ngraph_graph_id: int")
-    .SetIsStateful()
-    .SetShapeFn(shape_inference::ExplicitShape);
-
```

Comment:
we could remove `#include "tensorflow/core/framework/op_kernel.h"` from this file.

and maybe in other places too wherever applicable.